.github/workflows/triage.yml

```diff
@@ -6,11 +6,10 @@ on:
       - opened
 
 env:
-  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_TOKEN: ${{ secrets.PROJECT_BOARD_AUTOMATION }}
   ORGANIZATION: ethereum
-  REPOSITORY: solidity
-  PROJECT_NUMBER: 43
-  DRY_RUN: false
+  PROJECT_NUMBER: 27
+  DRY_RUN: true
```

Comment:
Or should this be false???

Reply:
See: https://github.com/ethereum/solidity/pull/13962#discussion_r1107553809